web/scripts/gitInfo.js

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable max-len */
-const { execSync } = require("child_process");
-const fs = require("fs");
-const path = require("path");
+import { execSync } from "child_process";
+import fs from "fs";
+import path from "path";
 
-const packageJson = require("../package.json");
+import packageJson from "../package.json" with { type: "json" };
```

Comment:
`with` can be replaced by `assert`

<img width="679" alt="Screenshot 2024-06-03 at 13 50 59" src="https://github.com/kleros/kleros-v2/assets/32412967/b1cc8f6c-d20e-4233-a548-41ee9722404c">